### PR TITLE
Remove leftover mentions of 3rdparty

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
Remove leftover mentions of `3rdparty` that should not be relevant any longer.

## Related Issue
Follows on from PR #41022 

## Motivation and Context
Clean up crud.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: build/build.xml [hunk 1]
```diff
@@ -11,7 +11,6 @@
```
Comment (DeepDiver1975) [inline build/build.xml:11]: This file can go fully ....

File: phpcs.xml [hunk 2]
```diff
@@ -5,7 +5,6 @@
 	<arg value="sp" />
```
Comment (DeepDiver1975) [inline phpcs.xml:5]: We are not using phpcs - I guess this file can go as well ...

Comment (phil-davis) [reply phpcs.xml:5]: IMO it is used for code checks of the acceptance tests.
The history (AFAIR) is that we enforced some code style on the acceptance tests in days before it was enforced more widely with the now-common owncloud code style.

I will have a look at removing the whole phpcs stuff separately.

File: build/build.xml (hunk 1 above)
Comment (phil-davis) [reply build/build.xml]: done

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by phil-davis** [comment]
https://github.com/owncloud/core/blob/master/apps/federatedfilesharing/templates/settings-personal-general.php
has stuff like:
```
script('federatedfilesharing', '3rdparty/gs-share/gs-share');
```
I don't see any 3rdparty folder inside federatedfilesharing.
It seems unrelated to files_external. But anyway, how does it work? Or does it work?

**Comment by phil-davis** [comment]
https://github.com/owncloud/core/blob/master/lib/private/AppFramework/Core/API.php
has `add3rdPartyScript` and `add3rdPartyStyle`
I suppose we don't remove or change those.

**Comment by DeepDiver1975** [comment]
> I don't see any 3rdparty folder inside federatedfilesharing.
> It seems unrelated to files_external. But anyway, how does it work? Or does it work?

This is referring back to the days when we had a 3rdparty repo linked in as git submodule.... I guess

gs-share is gnu share - which is dead - https://github.com/chimo/gs-share - we can remove this

**Comment by DeepDiver1975** [comment]
> has `add3rdPartyScript` and `add3rdPartyStyle`
> I suppose we don't remove or change those.

these are unused - let's keep it for the time being ...

**Pull request merged**